translations/translations.json

```diff
@@ -3110,7 +3120,17 @@
       "today": "today",
       "todayRecurring": "today, recurring",
       "onlyThisEvent": "Only this event",
-      "thisAndAllFutureEvents": "This and all future events"
+      "thisAndAllFutureEvents": "This and all future events",
+      "from": "from",
```

Comment:
I hope its okay but I slightly adjusted the translations and added German translations, perhaps you can check :)

Reply:
Thank you!